### PR TITLE
Sidebar menu hover effect

### PR DESCRIPTION
**Pull request by jyotiattri**
In this PR I have added some changes to the Sidebar menu to make it look like a part of the current theme.
Previous hover:
sidebar-hover
After change Hover:
sidebar-new-hover

### CODE REVIEW

**Comment by jyotiattri** [comment]
Hi, @zregvart 
Please let me know is there something to do in this PR.

File: antora-ui-camel/src/css/nav.css [hunk 1]
```diff
@@ -294,3 +294,10 @@ html.is-clipped--nav {
   background-position-y: 10px;
   width: 11rem;
 }
+
+aside.nav .nav-menu li a:hover,
+aside.nav .nav-menu .nav-text:hover {
+  color: #ed8225 !important;
+  text-decoration: underline;
+  text-decoration-color: #ed8225 !important;
+}
```
Comment (zregvart) [inline antora-ui-camel/src/css/nav.css]: This needs to be next to the .nav-menu on lines 118-146, needs to use the CSS variables we have for colors and needs to be without the `!important`.

Comment (jyotiattri) [reply antora-ui-camel/src/css/nav.css]: Ok, thanks for the input. I will fix it.

**Comment by zregvart** [comment]
@jyotiattri is there more work to be done here or are we going to close this?

**Comment by jyotiattri** [comment]
Please Close this.

**Pull request closed without merging**